packages/shared/src/components/auth/RegistrationForm.tsx

```diff
@@ -169,10 +170,11 @@ const RegistrationForm = ({
     );
     delete values?.['cf-turnstile-response'];
 
+    const requiresExperienceLevel = !isRecruiterOnboarding;
```

Comment:
We should really redo this whole flow to me more dynamic like freyja where a particular user can get any set of onboarding related questions or not. (future improvement)